src/components/icon/icon.test.tsx

```diff
@@ -60,7 +60,7 @@ function testIcon(props: PropsOf<OuiIcon>) {
       const onIconLoad = () => {
         component.update();
         expect(prettyHtml(component.html())).toMatchSnapshot();
-        resolve();
+        resolve(null);
```

Comment:
What prompted this change?

Reply:
Updating `Typescript` to v4.1.6 prompted this change (see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-1.html#resolves-parameters-are-no-longer-optional-in-promises), but I can use `Promise<void>` instead of `resolve(null)`, thanks for catching this